belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/MissingScopeProblem.java

```diff
@@ -99,4 +101,9 @@ public int hashCode() {
         return Objects.hash(super.hashCode(), requiredScopes);
     }
 
+    @Override
+    public Map<String, Object> getHttpResponseHeaders() {
+        return Collections.singletonMap(WWW_AUTHENTICATE, "Bearer error=\"insufficient_scope\"");
```

Comment:
scope(s) to indicate you might be missing multiple?

Reply:
The error code "insufficient_scope" is part of the spec.
https://datatracker.ietf.org/doc/html/rfc6750#section-3.1